src/assets/scripts/client/commandParser/argumentValidators.js

```diff
@@ -228,3 +229,20 @@ export const holdValidator = (args = []) => {
         }
     }
 };
+
+/**
+ * Checks that `args` has one value that corresponds to a valid squawk
+ *
+ * @function squawkValidator
+ * @param args {array}
+ * @return {string|undefined}
+ */
+export const squawkValidator = (args = []) => {
+    if (args.length !== 1) {
+        return ERROR_MESSAGE.SINGLE_ARG_LENGTH;
+    }
+
+    if (REGEX.TRANSPONDER_CODE.exec(args[0]) == null) {
```

Comment:
Those are just because of my general unfamiliarity with JS quirks/libs (which is btw good to keep an eye on if you review any JS from me :-)). I'll fix those.